app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html

```diff
@@ -38,11 +33,11 @@
 
 @pageTitlePrefix = {
     @if(state == AVAILABLE || state == UNAVAILABLE) {
-        @messages("dashboard.h1")
-    } else { @messages("dashboard.unknown.h1") }
+        @messages(s"dashboard.$serviceId.h1")
```

Comment:
line 34 is still a html rather than a String, which causes the new line break in the title, 
if you dont want to fix it as part of this PR, it fine but i think it can be fixed by 

```
@pageTitlePrefix = @{
    if(state == AVAILABLE || state == UNAVAILABLE) {
        messages(s"dashboard.$serviceId.h1")
    } else { messages(s"dashboard.$serviceId.unknown.h1") }
}
```